docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# HydraDX Code of Conduct
```

Comment:
I was thinking about this for quite some time and I'd like people to read this. This feels kind of OTT. I definitely wouldn't read through all of this when going to contribute to something as it's kind of clear once you read first few sentences. What do you think about keeping it very short but on point?